.github/workflows/dev-build-on-linter-commit.yml

```
@@ -0,0 +1,73 @@
+name: Scheduled check and build on new zlint commits
```

This isn't specific to Zlint.  I suggest:
```suggestion
name: Scheduled check and build on dependency updates
```